.github/workflows/publish-driver-adapters.yml

```
@@ -0,0 +1,80 @@
+name: Build and publish Prisma Driver Adapters
+
+concurrency: build-prisma-driver-adapters
+
+on:
+  # usually triggered via GH Actions Workflow in prisma/prisma repo
+  workflow_dispatch:
+    inputs:
+      enginesHash:
+        description: Engine commit hash to checkout for publishing
+        required: true
+      prismaVersion:
+        description: Prisma version to use for publishing
+        required: true
+      npmDistTag:
+        description: Npm Dist Tag to use for publishing
+        required: true
+        default: "latest"
+      dryRun:
+        description: 'Check to do a dry run (does not publish packages)'
+        type: boolean
+
+jobs:
+  build:
+    name: Build and publish Prisma Driver Adapters
+    runs-on: ubuntu-latest
+    steps:
+      - name: Print input
+        env:
+          THE_INPUT: "${{ toJson(github.event.inputs) }}"
+        run: |
+          echo $THE_INPUT
+
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.inputs.enginesHash }}
+
+      #
+      # Build
+      #
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "20.x"
```

```suggestion
      - uses: actions/setup-node@v3
        with:
          node-version: "20.x"
          registry-url: 'https://registry.npmjs.org/'
```

Example https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#publish-to-npmjs-and-gpr-with-npm